pages/[user].tsx

```diff
@@ -1,10 +1,14 @@
 import { GetServerSideProps } from "next";
 import Head from "next/head";
 import Link from "next/link";
-import prisma from "../lib/prisma";
+import prisma, {whereAndSelect} from "@lib/prisma";
```

Comment:
Codacy has a fix for the issue: [Replace `whereAndSelect` with `·whereAndSelect·`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
import prisma, { whereAndSelect } from "@lib/prisma";
```